scripts/run-pelorus-e2e-tests

```diff
@@ -205,11 +227,18 @@ trap 'cleanup_and_exit $?' INT TERM EXIT
 download_file_from_url "https://raw.githubusercontent.com/$demo_org/mig-demo-apps/$demo_branch/apps/todolist-mongo-go/pelorus/$ci_filename" "ci_values.yaml" "${DWN_DIR}"
 download_file_from_url "https://raw.githubusercontent.com/$demo_org/mig-demo-apps/$demo_branch/apps/todolist-mongo-go/mongo-persistent.yaml" "mongo-persistent.yaml" "${DWN_DIR}"
 
+oc get namespace "${PELORUS_NAMESPACE}" &> /dev/null
+if [ $? -eq 0 ]; then
+    echo "Namespace ${PELORUS_NAMESPACE} already exists. Delete it before running the script again."
+    # Add flag to pass other namespace name?
+    exit 1
+fi
 # Create namespace where pelorus and grafana, prometheus operators will get deployed
 oc create namespace "${PELORUS_NAMESPACE}"
 
 # Modify downloaded files
 sed -i.bak "s/your_org/$demo_org/g" "${DWN_DIR}/mongo-persistent.yaml"
+# TODO remove .bak?
```

Comment:
this creates an backup file with `.bak` extension, right? Is it necessary?

Reply:
If we do any changes on the non git file that could have been created by user a backup is recommended. In this example not really as the mongo-persistent.yaml is a file that was downloaded and used by this script. This backup would be important if the mongo-persistent.yaml was modified or created by the user.

Reply:
All the changes are done by `sed` in files stored in a temporary directory. What you think about removing them?

But we would to see what else needs to change (like line 244 would need to be deleted)